src/Razor/src/Microsoft.AspNetCore.Razor.VSCode/src/RazorLanguageServerClient.ts

```diff
@@ -236,11 +236,7 @@ export class RazorLanguageServerClient implements vscode.Disposable {
             args.push('--debug');
         }
 
-        this.serverOptions = {
```

Comment:
I'm not exactly sure what this did, but things seem to work fine using the simplified version of `this.serverOptions = { command, args };`

Reply:
Possibly it was just an older API, there have been a lot of additions since we first build the VSCode LS Client.